docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/dockerhub/index.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_label: Docker Hub
+description: How to configure Docker Hub webhooks with Kargo for automated artifact discovery.
+---
+
+# Docker Hub Webhook Receiver
+
+The Docker Hub Webhook Receiver responds to `push` events originating from Docker Hub repositories.
+
+In response to a `push` event, the receiver "refreshes" `Warehouse` resources subscribed to the corresponding Docker Hub repository.
+
+:::info
+"Refreshing" a `Warehouse` means enqueuing it for immediate reconciliation by the Kargo controller, which will attempt to discover new artifacts from all subscribed repositories.
+:::
+
+:::note
+Unlike GitHub, Docker Hub does **not** sign webhook payloads. Kargo uses a static token for basic validation.
+:::
+
+## Configuring the Receiver
+
+To enable webhook support for Docker Hub, you must configure a Kubernetes `Secret` and reference it in your `ProjectConfig`.
+
+The `Secret` must include a `dockerhub-secret` key under `stringData`. This value acts as a token for lightweight validation of incoming webhook requests.
```

Comment:
```suggestion
The `Secret` must include a `secret` key under `stringData`. This value acts as a token for lightweight validation of incoming webhook requests.
```

Reply:
I think there are much more accurate ways to describe this than a "token for lightweight validation." We could pretty easily explain that it's used in generating a hard-to-guess URL for the receiver. Search existing docs for wording you can reuse. It's mentioned in places like the working with Warehouses page.